src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,43 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Deadline object that is also a Task.
+ * Inherits from the Task class.
+ *
+ */
+public class Deadline extends Task {
+
+    private LocalDate time;
+    private final char type = 'D';
```

Comment:
Nice! I like how you specify the type.
However, I believe it could be static final since the type never changes for the class Deadline
Apart from that do use caps when define final constants
```suggestion
    private static final char TYPE_DEADLINE = 'D';
```

Reply:
Thanks for the review, will update it!